site/app/views/NavigationView.php

```diff
@@ -299,7 +299,7 @@ private function hasSubmitButton(Gradeable $gradeable): bool {
      */
     private function hasGradeButton(Gradeable $gradeable): bool {
         $now = $this->core->getDateTimeNow();
-        $date_limitation = $gradeable->getGradeStartDate() <= $now || $this->core->getUser()->getGroup() === User::GROUP_INSTRUCTOR || $this->core->getUser()->getGroup() === User::GROUP_FULL_ACCESS_GRADER;
+        $date_limitation = $gradeable->getGradeStartDate() <= $now || $this->core->getUser()->getGroup() === User::GROUP_INSTRUCTOR || $this->core->getUser()->getGroup() === User::GROUP_FULL_ACCESS_GRADER || $this->core->getUser()->getGroup() === User::GROUP_LIMITED_ACCESS_GRADER;
```

Comment:
Given its all numeric values under the hood, can simplify to a simpler less than or equal to:

```suggestion
        $date_limitation = $gradeable->getGradeStartDate() <= $now || $this->core->getUser()->getGroup() <= User::GROUP_LIMITED_ACCESS_GRADER;
```